apps/src/templates/textResponses/TextResponsesTable.story.jsx

```diff
@@ -33,27 +38,18 @@ const responses = [
   }
 ];
 
-export default storybook =>
-  storybook.storiesOf('TextResponsesTable', module).addStoryTable([
-    {
-      name: 'Text responses table',
-      story: () => (
-        <TextResponsesTable
-          responses={responses}
-          sectionId={sectionId}
-          isLoading={false}
-        />
-      )
-    },
-    {
-      name: 'Empty text responses table',
-      description: 'Displays an empty state message',
-      story: () => (
-        <TextResponsesTable
-          responses={[]}
-          sectionId={sectionId}
-          isLoading={false}
-        />
-      )
-    }
-  ]);
+const Template = args => <TextResponsesTable {...args} />;
+
+export const TextResponsesTableWithResponses = Template.bind({});
+TextResponsesTableWithResponses.args = {
+  responses: responses,
+  sectionId: sectionId,
```

Comment:
Since the 'sectionId' and 'isLoading' values are the same, we might want to pull them into the template to emphasize that the difference between these two stories is the response (or lack there of).